x/authz/simulation/operations_test.go

```diff
@@ -196,10 +197,11 @@ func (suite *SimTestSuite) TestSimulateExec() {
 
 	granter := accounts[0]
 	grantee := accounts[1]
-	a := banktypes.NewSendAuthorization(initCoins)
+	a, err := banktypes.NewSendAuthorization([]sdk.AccAddress{}, initCoins)
```

Comment:
```suggestion
	a, err := banktypes.NewSendAuthorization(nil, initCoins)
```

Why don't we use `nil`?